assets/js/job-submission.js

```diff
@@ -153,7 +153,7 @@ jQuery(document).ready(function($) {
 
 	// Listen for changes to the description field to clear validity
 	setTimeout( function() {
-		if ( typeof tinymce === "undefined" || tinymce.get( 'job_description' ) == null ) {
+		if ( !descriptionFieldIsPresent() ) {
```

Comment:
Should add a space after the `!` here.

Reply:
Thanks, fixed it in two places in f687665.